command/genesis/utils.go

```diff
@@ -126,7 +127,7 @@ func GetValidatorKeyFiles(rootDir, filePrefix string) ([]string, error) {
 }
 
 // ReadValidatorsByPrefix reads validators secrets on a given root directory and with given folder prefix
-func ReadValidatorsByPrefix(dir, prefix string) ([]*polybft.Validator, error) {
+func ReadValidatorsByPrefix(dir, prefix string, chainID int64) ([]*polybft.Validator, error) {
```

Comment:
This function is doing more stuff than its name represents now. It not only reads the validators by prefix but also initializes them for the manifest with the `kosk`. I would either:
- Rename this function to represent that.
- Keep it as it was and do the `kosk` in another step after this function is called.

Reply:
Code would be much more complicated if KOSK signature is generated outside, because in that case we should introduce another structure which contains everything `polybft.Validator` + `private key` - `signature`.

Can we just do first thing you have suggested - rename function? Do you have some suggestions for the new name?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1184/commits/da5bf28e62a21f569b8c14919c21f083cecc2c12